Alerting: Add warning in telegram contact point (#89397)
grafana/grafana: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-telegram.md

## Before you begin

### Telegram limitation

> Telegram messages are limited to 4096 UTF-8 characters. If you use a `parse_mode` other than `None`, truncation might result in an invalid message, and causing the notification to fail.

Reviewer: ```suggestion
Telegram messages are limited to 4096 UTF-8 characters. If you use a `parse_mode` other than `None`, truncation may result in an invalid message, causing the notification to fail.
```
